.github/workflows/luacheck.yml

```
@@ -7,4 +7,5 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
       - name: Luacheck
-        uses: lunarmodules/luacheck@master
\ No newline at end of file
+        run: |
+          docker run --rm -i -u luacheck --mount="type=bind,source=$(pwd),target=/repo" mineunit/luacheck
```

```suggestion
        uses: BuckarooBanzay/luacheck@minetest-builtin
```

Should work as there's action and builder defined in repo.
My docker image was mostly meant for easy local tests.

---

Actually probably wont work... only checked now and that action is pointing to prebuilt image.
Well, there's still another which uses docker builder:
```suggestion
        uses: S-S-X/minecheck@master
```